### PR TITLE
Fixed original user code in achievements.jade

### LINKED ISSUE #4696: Original User Badge
The "Original User" achievement does not display a badge, but one was made a long time ago by zoebeagle and apparently never added. It's here:

original_user_badge_by_zoebeagle

### PR DESCRIPTION
**Pull request by remmz0r**
Hi guys: fixes #4696

User: remmz0r fixed the originalUser icon on the profile page (i think)

First git pull request, please be gentle!

contact me if issues
- remmz0r

### CODE REVIEW

**Comment by crookedneighbor** [comment]
@remmz0r, I don't think you've pulled in the latest changes from the HabitRPG repo. Do you need assistance learning how to do that?

**Comment by remmz0r** [comment]
@crookedneighbor : I pulled from the develop branch just then and am about to try and put another pull request up. It ran fine on my system! But I couldnt figure out how to make a user an 'originalUser'

**Comment by Alys** [comment]
fixes #4696

**Comment by remmz0r** [comment]
@Alys - so can this be merged? I'm new to all this git stuff so excuse my ignorance. Added fixes to pull request. Seems the travis ci is only failing because of a database error.

**Comment by Alys** [comment]
We were all new to git at one stage. :)

We'll review your code some more to make sure it's okay, and if so, merge it. My local install is broken at the moment so I'm no use for that but there'll be others who will look at it. 

Note that it can sometimes take a few days to make time for a proper review (occasionally longer, depending on who's available and how busy we all are).

File: website/views/shared/profiles/achievements.jade [hunk 1]
```diff
@@ -135,6 +139,13 @@ div(ng-if=':: profile.achievements.originalUser || profile.achievements.veteran'
     small!=env.t('originalUserText')
```
Comment (crookedneighbor) [inline website/views/shared/profiles/achievements.jade]: You're very close to being done. Looks like you still have text related to the original user achievement in the veteran achievement. You can see the very end of it on this line. Remove lines 137-139 and you should be good to go. For reference, here's the whole code block with comments about which lines should be removed:

``` jade
div(ng-if=':: profile.achievements.veteran')
  .achievement.achievement-cake
  div(ng-if='::profile.achievements.veteran')
    h5=env.t('veteran')
    small=env.t('veteranText')
  div(ng-if='::profile.achievements.originalUser') // Remove me
    h5=env.t('originalUser') // Remove me
    small!=env.t('originalUserText') // Remove me
  hr

div(ng-if=':: profile.achievements.originalUser')
  .achievement.achievement-alpha
  div(ng-if='::profile.achievements.originalUser')
    h5=env.t('originalUser')
    small!=env.t('originalUserText')
  hr
```

**Comment by crookedneighbor** [comment]
@remmz0r Overall, this is looking great. If you'll remove the 3 lines I mentioned, I'll be happy to merge this in.

**Comment by remmz0r** [comment]
@crookedneighbor - Hi there! I think I just pushed a new version into it, but I am new to git and may have stuffed up with a merge accidentally... it's into my own branch I think! Still learning...

**Comment by remmz0r** [comment]
Pushing final code tested locally now.

**Comment by crookedneighbor** [comment]
@remmz0r No worries. 

It looks like you committed additional changes for a separate feature (adjusting guild links) to this branch. (I did the same thing when I first started using git, so it's no problem). Are you using the command line to do your git commands, or a program (such as the Github app)? Let me know, and I'll help you checkout a new branch for your changes, so your develop branch can more easily stay in sync with HabitRPG's develop branch.

When you get a chance, you may want [to read this article that outlines how branching works](https://help.github.com/articles/github-flow-in-the-browser/). If you have questions, please ask!

I'm going to merge in your changes related to the achievement and close this pull request, but you can still comment here and we'll get your repo all situated and back on track.

Also, this will count towards a contributor tier. Can you post your user id here? It can be found in [settings > api](https://habitrpg.com/#/options/settings/api)

**Comment by remmz0r** [comment]
@crookedneighbor - Thanks - yeah I'm very new at git. 

Here is my uid: 3fc19b18-342a-4993-a345-5a8cafe2b107

I've also posted changes on issue #4670 but I couldn't figure out how to remove the achievements.jade from the commit :(

I'll read the article in a moment after a coffee. Thanks for your time

**Comment by crookedneighbor** [comment]
Tier++

Don't worry about the achievements.jade commit. It's not going to hurt anything.

If you have questions, please ask them! I'd be happy to walk you through some git commands to help you get the hang of it. I just need to know how you're using it (command line versus a gui program).

**Comment by remmz0r** [comment]
Thanks for the tier!

I'm using tortoisegit but I've been using cmd line too just to learn a few things.

I managed to branch an 'archery' branch for another issue on the tracker but again, when I committed to archery the change in achievements.jade went through too

Not sure where I'm going wrong. I'm going to rebase my development tree and see if that helps.

**Pull request closed without merging**